project_recalculate/views/project_project_view.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <record id="edit_project_recalculate" model="ir.ui.view">
+            <field name="name">project.project.form</field>
+            <field name="model">project.project</field>
+            <field name="inherit_id" ref="project.edit_project"/>
+            <field name="arch" type="xml">
+
+                <button name="set_template" position="after">
+                    <button name="project_recalculate.wizard_project_recalculate"
+                            type="action"
+                            string="Recalculate project"
+                            states="open"
+                            context="{'project':id, 'project_name':name, 'calculation_type':calculation_type, 'date_start':date_start, 'date_end':date}"/>
```

Comment:
It's not clear to me how this works.
To use a context, I expected to see `context="{'default_project_id': id}"`, but it shouldn't be necessary since the wizard should use `active_id`.